gravatar/src/main/java/com/gravatar/services/Result.kt

```diff
@@ -0,0 +1,30 @@
+package com.gravatar.services
+
+/**
+ * Class representing the result of a network operation.
+ */
+public sealed class Result<T, E> {
```

Comment:
🤩 Looks great! As at the moment `E` is always `ErrorType`, we can remove that. But I'm totally fine with the current implementation.

Reply:
Yes, the idea is to have a generic Result class that could be used for anything. That's also why I updated the `GravatarListener` class, so it's just the same kind of interface but with a different taste.